awx/api/serializers.py

```diff
@@ -4685,10 +4685,10 @@ class InstanceSerializer(BaseSerializer):
 
     class Meta:
         model = Instance
-        read_only_fields = ('uuid', 'hostname', 'version')
+        read_only_fields = ('uuid', 'hostname', 'version', 'is_containerized')
```

Comment:
If you remove https://github.com/ansible/awx/pull/4189/files#r331524755, you can lose this, too.